pyproject.toml

```diff
@@ -196,3 +196,11 @@ lint.pydocstyle.convention = "google"
 
 [tool.coverage.run]
 concurrency = [ "gevent" ]
+
+[tool.pyright]
```

Comment:
sidefix: make vscode's language servers not crash under the load of eval logs in the default location.